packages/js/product-editor/src/blocks/product-fields/name/edit.tsx

```diff
@@ -55,14 +54,16 @@ export function NameBlockEdit( {
 		useState( false );
 
 	const productId = useEntityId( 'postType', 'product' );
-	const product: Product = useSelect( ( select ) =>
-		// eslint-disable-next-line @typescript-eslint/ban-ts-comment
-		// @ts-ignore
-		select( 'core' ).getEditedEntityRecord(
-			'postType',
-			'product',
-			productId
-		)
+	const product: Product = useSelect(
+		( select ) =>
+			// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+			// @ts-ignore
+			select( 'core' ).getEditedEntityRecord(
```

Comment:
In such cases you should

`import { store as coreStore } from '@wordpress/core-data`

and use it in here instead of raw string and that should resolve the ts error:

```suggestion
			select( coreStore ).getEditedEntityRecord(
```

Reply:
Actually I don't recall adding that ignore directive. I think I might of been testing something and got committed. But nonetheless I tried using the import instead of the string but still got TS errors.

Reply:
I added a comment that it will be fixed in the data packages [000fa9a](https://github.com/woocommerce/woocommerce/pull/54108/commits/000fa9a29330357354b63cd8e620cd267ecdeaf0)